packages/firestore/src/api/database.ts

```diff
@@ -552,6 +553,61 @@ export class WriteBatch
   }
 }
 
+class UntypedFirestoreDataConverterAdapter<U>
```

Comment:
Can you add some JSDoc for this class? You can likely summarize your PR description.

Reply:
Done.